.github/workflows/cf-worker-example-test.yml

```diff
@@ -0,0 +1,45 @@
+name: Run CF Worker Example App
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on:
+      labels: ubuntu-latest-4-core
+    strategy:
+      matrix:
+        node-version: [18.x]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'yarn'
+      - name: Install protoc for protobuf compilation
+        run: sudo apt-get install -y protobuf-compiler
```

Comment:
Do we still need this? I think I took it out in the release workflow and didn't see any problems